### ISSUE TITLE
Add screencasts to the main page

### ISSUE THREAD

**Issue by rominf** [fish-shell/fish-site#17]
It really attracts people.

I've prepared fish completions for `salt` https://github.com/saltstack/salt/pull/15310 and prepared screencast (gif, using Byzanz). It's a good way to demonstrate features of fish (picture itself says much less).

I think it could look like this: http://dimsemenov.com/plugins/royal-slider with 3-4 short screencast showing autocompletion, multiline editing, fish web configuration.

**Comment by floam** [comment]
I think screencasts would be good. Perhaps we could self-host http://asciinema.org screencasts? It uses HTML5/js. It's rather nice not having any large images/movies for people to download, and having them render with real fonts at whatever size their browser wants at native resolution.

**Comment by ridiculousfish** [comment]
Yes! If anyone wants to put these together I'll be glad to upload them.

**Comment by zanchey** [comment]
Possible items:

- autosuggestion
- syntax highlighting
- on-the-fly indentation
- abbreviations
